.github/workflows/ephemeral-env.yml

```diff
@@ -79,55 +79,51 @@ jobs:
           })
           core.setFailed(errMsg)
 
-  docker_ephemeral_env:
-    needs: config
-    if: needs.config.outputs.has-secrets
-    name: Push ephemeral env Docker image to ECR
+  ephemeral-docker-build:
+    needs: ephemeral_env_comment
+    name: docker-build
     runs-on: ubuntu-latest
-
     steps:
-      - name: "Download artifact"
-        uses: actions/github-script@v3.1.0
+      - name: Get Info from comment
+        uses: actions/github-script@v3
+        id: get-pr-info
         with:
           script: |
-            const artifacts = await github.actions.listWorkflowRunArtifacts({
+            const request = {
                 owner: context.repo.owner,
                 repo: context.repo.repo,
-                run_id: ${{ github.event.workflow_run.id }},
-            });
-
-            core.info('*** artifacts')
-            core.info(JSON.stringify(artifacts))
-
-            const matchArtifact = artifacts.data.artifacts.filter((artifact) => {
-              return artifact.name === "build";
-            });
-
-            if (!matchArtifact.length) {
-              return core.setFailed("Build artifacts not found");
+                pull_number: ${{ github.event.issue.number }},
             }
+            core.info(`Getting PR #${request.pull_number} from ${request.owner}/${request.repo}`)
+            const pr = await github.pulls.get(request);
```

Comment:
To fetch the last SHA from the PR where the comment was made, more details here: https://github.com/actions/checkout/issues/331